### PR TITLE
RSS Feed

### LINKED ISSUE #139: add RSS feed to blog
# Bounty

currently the blog does not expose an RSS feed. hence it is not possible to subscribe to posts or use other readers. 

without RSS podcasting is also not possible using the blog.

## Scope

- enable the generation of an RSS during build of the website
- include all blog articles in the RSS feed
- (optional) if the blog contains a media file, include in RSS feed

## Deliverables

- create `rss.xml.js` in `/blog`
- test subscription with Feedly, or some other mobile feed reader

## Gain for the project

- more reach through subscriptions
- ability to start podcast

### Publicly visible delivery

rss feed

## Roles
bounty gardener: @johannbarbie  / 10%
bounty worker: @sunify  / 70%
bounty reviewer: name / share

### PR DESCRIPTION
**Pull request by sunify**
Resolves #139

### CODE REVIEW

File: src/routes/rss.xml.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+import { siteHost } from "../server";
+
+function makeAuthor(author) {
+  const result = {
+    name: author.name,
+    link: author.link
+  };
+
+  if (author.twitter) {
+    result.link = `https://twitter.com/${author.twitter}`;
```
Comment (johannbarbie) [inline src/routes/rss.xml.js]: this produces `<![CDATA[ @johba (https://twitter.com/@johba) ]]>`, i guess it should be without the `@`

Comment (sunify) [reply src/routes/rss.xml.js]: 🤦‍♂

File: src/routes/rss.xml.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import RSS from "rss";
-import { getPosts } from "./blog/_posts.js";
-import { siteHost } from "../server";
+import { getPosts } from "../blog/_posts.js";
+import { siteHost } from "../../server";
```
Comment (sunify) [inline src/routes/rss.xml.js]: Wow :-) Why?

Comment (johannbarbie) [reply src/routes/rss.xml.js]: ups, sry, i got confused while merging around 🤕

**Pull request merged**